main/src/library/Process.flix

```diff
@@ -73,7 +93,48 @@ mod Process {
     import java.lang.ProcessBuilder
     import java.lang.Runtime
     import java.io.{File => JFile}
+    import java.io.{InputStream => JInputStream}
+    import java.io.{OutputStream => JOutputStream}
     import java.util.concurrent.{TimeUnit => JTimeUnit}
+    use Process.StdErr.StdErr
+    use Process.StdIn.StdIn
+    use Process.StdOut.StdOut
+
+    pub enum StdIn(JOutputStream)
+
+    instance Writer[StdIn] {
+
```

Comment:
Remove linebreak

Reply:
Just to keep things tight. I think you can do the same for the other two instances.